### PR TITLE
data: ArjanDhakal

### PR DESCRIPTION
**Pull request by arjandhakal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5321"></a>

### CODE REVIEW

File: data/arjandhakal.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name":"Arjan Dhakal",
+  "type": "personal",
+  "bio": "Fiddling with Typescript and Rust",
+  "avatar": "https://github.com/arjandhakal.png",
```
Comment (imbeshat) [inline data/arjandhakal.json:5]:
```suggestion
```

File: data/arjandhakal.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "links":[
+    {
+      "name":"Github",
+      "url": "https://github.com/arjandhakal",
+      "icon": "FaGithub"
+    },
+    {
+      "name":"Blog Articles",
+      "url": "https://arjandhakal.hashnode.dev",
+      "icon": "FaBlogger"
```
Comment (imbeshat) [inline data/arjandhakal.json:15]:
```suggestion
      "icon": "SiHashnode"
```

Comment (imbeshat) [reply data/arjandhakal.json:15]: As the blog site is Hashnode. So, I think hashnode icon will be better.

**Review by imbeshat — CHANGES_REQUESTED** [review]

File: data/arjandhakal.json (hunk 2 above)
Comment (arjandhakal) [reply data/arjandhakal.json]: Ah, good catch. Didn't know we had a icon for it.

File: data/arjandhakal.json (hunk 1 above)
Comment (imbeshat) [reply data/arjandhakal.json:5]: ```avatar``` is not required.

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**